src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+
+import InlineSvg from 'app/components/inlineSvg';
+import TextCopyInput from 'app/views/settings/components/forms/textCopyInput';
+
+interface Props {
+  name: string;
+  version: string;
+  culture: string;
+  publicKeyToken: string;
+  filePath: string;
+}
+
+const Assembly: React.FC<Props> = ({
+  name,
+  version,
+  culture,
+  publicKeyToken,
+  filePath,
+}) => {
+  return (
+    <AssemblyWrapper>
+      <Icon src="icon-return-key" />
+      <AssemblyInfo>
+        <Caption>Assembly:</Caption>
```

Comment:
Should these labels/captions be wrapped in `t()`?

Reply:
I'd say an assembly is an assembly in any language.

Here are Microsoft docs in Portuguese 

https://docs.microsoft.com/pt-br/dotnet/standard/assembly/

Reply:
Wrapped them in t`()` just to be sure.

Reply:
😢

Reply:
![image](https://user-images.githubusercontent.com/1633368/71915357-de632780-317b-11ea-9d20-4eaa107242ab.png)
I'd say we don't add this, if this gets translate it'll be super confusing.

Reply:
fwiw we haven't updated our localization in a long time, it doesn't get auto translated, I think we should get in the habit of always using `t()` for when we do decide to fix our localization though.

Reply:
Fair enough. I'm just hoping we don't put it here (maybe needs a comment to clarify it's intentional?) this English word is the name of the thing in any language. If it translates it won't make sense.